config/mcu_definitions/main/BTT_SKR_Pico_v1.0.cfg

```diff
@@ -0,0 +1,24 @@
+[board_pins mcu_manufacturer]
+aliases:
+    MCU_XM_STEP=gpio11 , MCU_XM_DIR=gpio10 , MCU_XM_ENABLE=gpio12 ,
+    MCU_YM_STEP=gpio6 , MCU_YM_DIR=gpio5  , MCU_YM_ENABLE=gpio7 ,
+    MCU_ZM_STEP=gpio19  , MCU_ZM_DIR=gpio28  , MCU_ZM_ENABLE=gpio2 ,
+    MCU_EM_STEP=gpio14  , MCU_EM_DIR=gpio13  , MCU_EM_ENABLE=gpio15 ,
+    MCU_TMCUART=gpio9 , MCU_TMCTX=gpio8  ,
+
+    MCU_XSTOP=gpio4   , MCU_YSTOP=gpio3  , MCU_ZSTOP=gpio25   ,
+    MCU_E0STOP=gpio16 , 
+
+    MCU_HE0=gpio23 ,
+    MCU_BED=gpio21 ,
```

Comment:
This is `MCU_HE` and `MCU_HB`. Also you can probably add `MCU_LASER=gpio20`

Reply:
Sorry LASER is a shared pin with FAN3. So as Klippain is dedicated to 3D printer primarly, I think it's better to use FAN3 for this.